__tests__/app.test.js

```diff
@@ -75,6 +75,26 @@ describe("News app",()=>{
                 expect(body.msg).toEqual("Article not found")
             })
         })
+        test("REFACTOR: status:200, responds with comment count",()=>{
+            const article_id = 9;
+            return request(app)
+            .get(`/api/articles/${article_id}`)
+            .expect(200)
+            .then(({body}) => {
+                const article = body.article;
+                expect(article.comment_count).toBe("2")
+            })
+        })
+        test("REFACTOR: status:200, second test responds with comment count",()=>{
```

Comment:
there seems to be no difference between these tests 🤔 
For s second one I think it'd be better to do article 2, it doesn't have and comments and wouldn't work if you didn't have the `LEFT JOIN` in your query